packages/osc-ui/src/components/Accordion/accordion.scss

```
@@ -0,0 +1,239 @@
+            color: var(--color-quaternary);
+        }
+
+        &:focus-visible .c-accordion__icon {
+            box-shadow: 0 0 0 0.5rem var(--color-tertiary), 0 0 0 0.75rem var(--color-primary);
+        }
+
+        &[data-state="open"] .c-accordion__icon--plusminus > :first-child {
+            opacity: 0;
+            visibility: hidden;
+        }
+
+        &[data-state="open"] .c-accordion__icon--chevron {
+            transform: rotate(180deg);
+        }
+
+        /* stylelint-disable-next-line plugin/selector-bem-pattern */
+        > span {
+            grid-column: 1 / 2;
+        }
+    }
+
+    &__content {
+        max-width: 64ch;
+        overflow: hidden;
+        font-size: var(--accordion-content-font-size);
+        font-weight: 300;
+
+        &[data-state="open"] {
+            animation: slide-down var(--accordion-transition-duration) var(--accordion-transition-timing-func);
+        }
+
+        &[data-state="closed"] {
+            height: 0;
+            animation: slide-up var(--accordion-transition-duration) var(--accordion-transition-timing-func);
+        }
+    }
+
+    &__text {
+        padding: var(--font-scale-milli) 0;
+    }
+
+    // *----------------------------------*/
+    //  Variants
+    // --primary not included as it would be the default
+    // *----------------------------------*/
+    &--secondary {
+        --accordion-icon-size: 1.6em;
+        --accordion-trigger-font-size: var(--font-scale-centi);
+
+        // Add new column to the grid so we can centre the text accurately
+        --accordion-trigger-grid-columns: minmax(0, var(--accordion-icon-size)) 1fr auto;
+
+        .c-accordion__item {
+            border: var(--accordion-item-border);
+
+            &:not(:last-child) {
+                padding-bottom: 0;
+                border-bottom: none;
+            }
+        }
+
+        .c-accordion__trigger {
+            padding:
+                var(--space-scale-epsilon)
+                var(--space-scale-delta);
+            text-align: center;
+            text-transform: uppercase;
+            letter-spacing: 0.188em;
+
+            /* stylelint-disable-next-line plugin/selector-bem-pattern */
+            > span {
+                grid-column: 2 / 3;
+            }
+
+            @include mq(mq-token("mob")) {
+                gap: var(--space-scale-zeta);
+            }
+
+            &[data-state="open"] {
+                color: var(--color-quaternary);
+            }
+        }
+
+        /* stylelint-disable-next-line no-descending-specificity */
+        .c-accordion__icon {
+            grid-column: 3 / -1;
+            color: currentcolor;
+        }
+
+        .c-accordion__content {
+            max-width: 100%;
+        }
+
+        .c-accordion__text {
+            padding: 0 var(--space-scale-delta) var(--space-scale-delta);
+        }
+    }
+
+    &--tertiary {
+        --accordion-trigger-font-size: #{rem(24)}; // design has this fixed across breakpoints
```

Just a thought but would it be worth calculating the value of this from the base-font-size?
```suggestion
        --accordion-trigger-font-size: #{rem(($base-font-size * 0.75) * 2)}
```

---

Ooo yeah that's a good shout